src/style.css

```diff
@@ -2,0 +1,27 @@
+.stars {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  padding: 10px;
+  border-radius: 5px;
+}
+
+.stars__row {
+  display: flex;
+  gap: 5px;
+}
+
+.stars__star {
+  width: 30px;
+  height: 30px;
+  background-size: cover;
+  background-image: url('../src/images/star-icon.svg');
```

Comment:
The relative path to the background image should be updated to reflect the CSS file's location. Since the CSS file is in `src/`, the correct path would be `images/star-icon.svg` without the `../src/` prefix.